collections/Laud_Misc/MS_Laud_Misc_626.xml

```diff
@@ -39,20 +42,69 @@
                </msIdentifier>
                <msContents>
                   <msItem n="1" xml:id="MS_Laud_Misc_626-item1">
+                     <locus>(fols. 1r–190v)</locus>
                      <author key="person_69105608">Hugutio of Pisa</author>
                      <title key="work_2175">Liber deriuationum</title>
+                     <msItem>
+                        <locus>(fol. 1r)</locus>
+                        <note>Prologue</note>
+                        <rubric>Incipit prologus super derivationes maiores</rubric>
+                        <incipit> Cum nostri protoplasti suggestiva prevaricatione. <sic>dyabolus</sic> humanum genus a sue dignitatis culmine quam longe <choice><sic>deiecerit</sic> <corr>deciderit</corr></choice> </incipit>
+                        <explicit>a verbo augmenti noftre assertionis auspicium sorciamur</explicit>
+                        <finalRubric>Explicit prologus</finalRubric>
+                        <note>The author is referred to as ‘ … patria Pisanus nomine <sic>Huicio</sic> …’. </note>
+                        </msItem>
+                     <msItem>
+                        <locus>(fols. 1r–190v)</locus>
+                        <rubric>Incipiunt derivationes <sic>Huicii</sic> </rubric>
+                        <incipit>Augeo es. auxi. ctum. amplificare id est augmentum dare</incipit>
+                        <explicit defective="true">unde ille homo dicitur zelotipus qui suspectionem habet de</explicit>
+                     </msItem>
+                     <note>A leaf is lost (apparently excised) before fol. 1 (with loss of a table of contents(?)); for other missing leaves see below, Collation. Some of these losses are signalled in marginal pencil notes by a 19th- or early 20th-century century reader, not always correctly (e.g. there is no leaf missing after fol. 101).</note>
                      <textLang mainLang="la">Latin</textLang>
+                     <!-- text online: https://dama.dantenetwork.it/fileadmin/texts/Uguccione_Derivationes.xml
+                      = 
+                     https://dama.dantenetwork.it/index.php?id=17&workSign=Uguccione_Derivationes&L=0
+                     -->
+                  </msItem>
+                  <msItem>
+                     <note>One gloss in German, fol. 82r: <q>auis preda<damage/> scilicet valcke</q></note>
```

Comment:
Is `<gap/>` meant instead of `<damage/>`?

Reply:
text lost due to cropping